docs/content.zh/docs/dev/table/data_stream_api.md

```diff
@@ -596,13 +596,13 @@ pipeline or a statement set:
 
 ```java
 // execute with explicit sink
-tableEnv.from("InputTable").executeInsert("OutputTable")
+tableEnv.from("InputTable").insertInto("OutputTable").execute()
```

Comment:
Sorry for the late response. @leonardBang @zoucao 
Looks good to me on the whole. However, minor confusion from my limited read： Should the Java code here have an ending symbol ‘;’ ?

Reply:
You're right @RocMarshal , look like it's an existed doc bug we should fix, @zoucao Would you like to append one commit to check the java code in all Flink docs and fix them ?

Reply:
Hi @leonardBang, I'm glad to do this, and thanks @RocMarshal for proposing it. I'm sorry for the late reply.
I have checked all the java code and found the missing symbol ‘;’ existing in our java code, should I fix it in this pr or create another pr called [hotfix]? I think the ending symbol ‘;’ for other java codes is not closely related to this pr.

Reply:
Both appending one commit `[hotfix][docs]` or opening another PR  `[hotfix][docs]` make sense to me, it depends on your habit.

Reply:
Well, I will append one commit to fix it in this pr as far as possible.

Reply:
Hi, @RocMarshal , could you help me confirm whether the ';' should be added for one line java code,
for example :
not added originally：

> - Start from the earliest available message in the topic.
>   ```java
>   StartCursor.earliest()
>   ```
> - Start from the latest available message in the topic.
>   ```java
>   StartCursor.latest()
>   ```
> - Start from a specified message between the earliest and the latest.
>   Pulsar connector would consume from the latest available message if the message id doesn't exist.
> 
>   The start message is included in consuming result.
>   ```java
>   StartCursor.fromMessageId(MessageId)
>   ```
added originally：

> 
> ```java
> // discover new partitions per 10 seconds
> PulsarSource.builder()
>         .setConfig(PulsarSourceOptions.PULSAR_PARTITION_DISCOVERY_INTERVAL_MS, 10000);
> ```
All were chosen from pulsar.md.

Reply:
Yes, IMO, It should be added.

Reply:
Hi, @RocMarshal, take a look, plz